### DISCUSSION TITLE
It a short note of thanks.

### DISCUSSION THREAD

**Discussion by Xyncgas** [nlohmann/json#2372] [General]
Your library gave my life a easier target as a programmer to achieve something with the things I can do now. Like getting a new toy for a children or a new color pencil set for a young teenager who would takes a interest in drawing. I too couldn't figure out most the library of how to use them and all the building and cmake and git and stuff, because I am playing with c++ and it's the only thing I know right n, which I take great interest and let it guide me, I know it's important to follow my passion but at the same time I don't wanna stop to train myself to get to know all the popular tool chains people are using. I feel like there's nothing wrong, for me to move forward like this, and I am sure people have great reasons to make their library hard to use too, as I am starting to find out how seperate translation unit might save compile time, still a header only library makes my coding experience so much cooler and I can now do more things!

Thanks for helping me and lending me your effort for this wonderful gifts, I will remember what you did and what it meant for me as a wish to return people this favor in the future.

I know sqlite has this header only (well with a couple .c source) library that's really accessible too, and I like these library way better than something like libcurl because they are easier to use, which I just can't find a friend in libcurl developer's heart to let me build a program using static linked library, while if it's a header only library it would be quite easy.

Sometime the convienience isn't even that big too I feel like if people use cmake and stuffs to save compile time by essentially having different translation units.

I love writing codes and I can do a lot cool things with json while learning about programming too, this library and related people's effort on this project is like making me feel lifted and cared, I wanna now say thanks again, and even though I could've probably written a json library myself, I enjoy every bit of the reliability and having someone else to maintain it and keeping it high quality while I still don't know about lot programming stuffs and can really have fun doing something rn too! And one day perhaps I can contribute to this project by making my pull request.

**Comment by nlohmann** [comment]
Thanks a lot! If you want to support me and my work on this project, you can become a sponsor on GitHub (https://github.com/sponsors/nlohmann), or use Paypal (http://paypal.me/nlohmann).
